### PR TITLE
accept "default" as "ok response" fallback

### PR DESCRIPTION
**Pull request by jkone27**
tries to fix: https://github.com/fsprojects/SwaggerProvider/issues/219. 
@sergey-tihon

### CODE REVIEW

**Comment by sergey-tihon** [comment]
would be also nice to add your schema to test schemas - https://github.com/fsprojects/SwaggerProvider/tree/master/tests/SwaggerProvider.ProviderTests/Schemas/v3

and add test like this https://github.com/fsprojects/SwaggerProvider/blob/master/tests/SwaggerProvider.ProviderTests/v3/Swagger.I0173.Tests.fs to check that TP works on this schema and manually check return type before merge/release

**Comment by jkone27** [comment]
added a sample and tests, thanks!

**Comment by jkone27** [comment]
Does it look good now?

**Comment by sergey-tihon** [comment]
@jkone27 does it already work for you?

File: src/SwaggerProvider.DesignTime/v3/OperationCompiler.fs [hunk 1]
```diff
@@ -156,11 +156,7 @@ type OperationCompiler(schema: OpenApiDocument, defCompiler: DefinitionCompiler,
         let retMimeAndTy =
             let okResponse =
                 operation.Responses
-                |> Seq.tryFind(fun resp -> 
-                    resp.Key = "200" 
-                    || resp.Key.StartsWith("20")
-                    || resp.Key = "default"
-                )
+                |> Seq.tryFind(fun resp -> resp.Key = "200" || resp.Key.StartsWith("20") || resp.Key = "default")
```
Comment (jkone27) [inline src/SwaggerProvider.DesignTime/v3/OperationCompiler.fs:159]: ah thanks, i tried to run fantomas format from vscode in my github codespace but was erroring and i didn't get what was wrong

File: tests/SwaggerProvider.ProviderTests/SwaggerProvider.ProviderTests.fsproj [hunk 2]
```diff
@@ -23,6 +23,7 @@
     <Compile Include="v3\Swagger.PetStore.Tests.fs" />
     <Compile Include="v3\Swagger.I0173.Tests.fs" />
     <Compile Include="v3\Swagger.I0181.Tests.fs" />
+    <Compile Include="v3\Swagger.I0219.Tests.fs" />
```
Comment (jkone27) [inline tests/SwaggerProvider.ProviderTests/SwaggerProvider.ProviderTests.fsproj:26]: oh right, forgot to include it, thanks!

**Comment by jkone27** [comment]
thanks for patching my changes @sergey-tihon 🎸

**Comment by sergey-tihon** [comment]
@jkone27 thank you for the fix, released in [v2.0.0-beta8](https://www.nuget.org/packages/SwaggerProvider/2.0.0-beta8)

**Pull request merged**